pytorch_lightning/accelerators/tpu_backend.py

```diff
@@ -67,7 +67,8 @@ def teardown(self):
         last_path = self.mp_queue.get()
 
         # transfer back the best path to the trainer
-        self.trainer.checkpoint_callback.best_model_path = best_path
+        if self.trainer.checkpoint_callback is not None:
+            self.trainer.checkpoint_callback.best_model_path = best_path
```

Comment:
how about we just merge this one line fix into master? This would close the issue before release and you can still figure out the hard testing part for TPU in the other PR that's blocking this. 
cc @Borda

Reply:
sounds good to me...

Reply:
That's sounds good 👍